### PR TITLE
Todo List

### PR DESCRIPTION
**Pull request by Fanger53**
## This project has the next features:
-you can create a project. you can create and delete this project
-each project has many tasks. you can edit and delete these tasks

it was created with:
- webpack
- javascript
- html
- scss

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name: Linters
```
Comment (addod19) [inline .github/workflows/linters.yml:1]: Kindly use the right linter configuration which is used by students of Microverse. You can find it [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml) :muscle:

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off"
```
Comment (addod19) [inline .eslintrc.json:17]: Do not make any changes to the linter files....It should be exactly like this [file](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) :+1:

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @Fanger53 and @Nexch, congratulations on this project :1st_place_medal: The application works as expected, a user can create a project, delete a project, create a task, edit a task, delete a task, etc :muscle: 

The only issue we have is the linter configuration files, let us use the recommended files for all Microverse students

- The DOM manipulation and application logic should be in a separate module. It would be professional to have a module file for DOM manipulation and one for the logic so that it would be easy to test and debug in case there is a problem


Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 
Nice work making the requested changes.
Let's make some changes before we can merge this PR.
- [ ] Separate logic from DOM manipulation. This was already pointed out by the previous reviewer.


### Martin Afani
#### Slack: whiz25

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hi @Fanger53 and @Nexch  ! :wave:

#### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

#### You may merge, good luck with your next projects! :rocket:
#### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**